django/contrib/admin/models.py

```diff
@@ -25,8 +25,18 @@ class LogEntryManager(models.Manager):
     use_in_migrations = True
 
     def log_actions(
-        self, user_id, queryset, action_flag, change_message="", *, single_object=False
+        self,
+        user_id,
+        queryset,
+        action_flag,
+        change_message="",
+        action_by_repr="",
+        *,
+        single_object=False,
     ):
+        if not (user_id or action_by_repr):
+            raise ValueError("user_id or action_by_repr must be provided")
```

Comment:
This may not be strictly necessary, but I think logging an action without any indication who or what performed it would be strange.